pdctl/command/hot_command.go

```diff
@@ -21,8 +21,9 @@ import (
 )
 
 const (
-	hotRegionsPrefix = "pd/api/v1/hotspot/regions"
-	hotStoresPrefix  = "pd/api/v1/hotspot/stores"
+	hotRegionsPrefix     = "pd/api/v1/hotspot/regions"
+	hotReadRegionsPrefix = "pd/api/v1/hotspot/readregions"
```

Comment:
Maybe `regions/write` and `regions/read`.

Reply:
> "pd/api/v1/hotspot/regions"

this is for write, maybe
> "pd/api/v1/hotspot/read"

this is for read ?